app/assets/stylesheets/activity.scss

```diff
@@ -0,0 +1,17 @@
+.activity-cell {
```

Comment:
Looks like these get overriden by default styles. Can probably be removed

Reply:
Pol is fixing the styles :)